pkg/bootstrap/types.go

```diff
@@ -8,7 +8,6 @@ type KuryrBootstrapResult struct {
 	ServiceSubnet            string
 	PodSubnetpool            string
 	WorkerNodesRouter        string
-	WorkerNodesSubnet        string
```

Comment:
I'm assuming you're removing this as you now import "github.com/gophercloud/gophercloud/openstack/networking/v2/subnets"

Reply:
I'm removing this because we no longer need CNO to pass detected *main* worker nodes subnet to the Kuryr configuration. It's because we're now detecting that directly in kuryr-controller by watching the Machine objects in order to make sure we're able to react to new Machines being spawned on separate subnets.

Reply:
BTW - CNO still needs to **detect** the _main_ worker nodes subnet (in a different way though, now it needs to list Machines due to changes in the installer) in order to have its CIDR and create correct SG rules for Kuryr. The ShiftOnStack assumption is that for nodes created on separate subnets it's the users responsibility to create the SG and SG rules matching that CIDR. That's why CNO only needs one _main_ worker nodes subnet.